File: src/main/java/com/heybys/oddments/fooddelivery/persistence/shop/ShopJpaRepository.java
package com.heybys.oddments.fooddelivery.persistence.shop;

import org.springframework.data.jpa.repository.JpaRepository;

import com.heybys.oddments.fooddelivery.domain.shop.Shop;
import com.heybys.oddments.fooddelivery.domain.shop.ShopId;

interface ShopJpaRepository extends JpaRepository<Shop, ShopId> {}

File: src/main/kotlin/com/heybys/oddments/fooddelivery/persistence/shop/ShopJpaRepository.kt
package com.heybys.oddments.fooddelivery.persistence.shop

import com.heybys.oddments.fooddelivery.domain.shop.Shop
import com.heybys.oddments.fooddelivery.domain.shop.ShopId
import org.springframework.data.jpa.repository.JpaRepository

internal interface ShopJpaRepository : JpaRepository<Shop, ShopId>
